<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmAcercaD</class>
 <widget class="QDialog" name="frmAcercaD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>440</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Acerca del GR-lida</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lb_imagenlogo">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../res/gr-lida.qrc">:/images/gr-lida_logo.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <property name="horizontalSpacing">
      <number>8</number>
     </property>
     <item row="0" column="0" rowspan="3">
      <widget class="QLabel" name="lb_icono">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../res/gr-lida.qrc">:/img32/gr-lida.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>Versión del GR-lida:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="lb_version">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Diseñado y programado por:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Monthy</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Lista de compañías de juegos creada por:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Dantoine</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QLabel" name="lb_byOS">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item>
        <widget class="QLabel" name="lb_Dbximg">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../res/gr-lida.qrc">:/img24/cat/dosbox.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_Dbx">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>DOSBox</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_Svmimg">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../res/gr-lida.qrc">:/img24/cat/scummvm.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_Svm">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>ScummVM</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_Vdmsimg">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../res/gr-lida.qrc">:/img24/cat/vdmsound.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_Vdms">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>VDMSound</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="QTabWidget" name="tabAcercaD">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_1">
        <attribute name="icon">
         <iconset resource="../res/gr-lida.qrc">
          <normaloff>:/img16/informacion.png</normaloff>:/img16/informacion.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Info</string>
        </attribute>
        <layout class="QVBoxLayout">
         <property name="margin">
          <number>4</number>
         </property>
         <item>
          <widget class="QTextBrowser" name="txtAcercaD">
           <property name="openExternalLinks">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="icon">
         <iconset resource="../res/gr-lida.qrc">
          <normaloff>:/img16/idiomas.png</normaloff>:/img16/idiomas.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Traducciones</string>
        </attribute>
        <layout class="QVBoxLayout">
         <property name="margin">
          <number>4</number>
         </property>
         <item>
          <widget class="QTreeWidget" name="twTraductores">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <weight>50</weight>
             <bold>false</bold>
             <strikeout>false</strikeout>
             <stylestrategy>PreferDefault</stylestrategy>
            </font>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
           <property name="uniformRowHeights">
            <bool>true</bool>
           </property>
           <property name="itemsExpandable">
            <bool>false</bool>
           </property>
           <property name="allColumnsShowFocus">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string>Idioma</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Autor</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="icon">
         <iconset resource="../res/gr-lida.qrc">
          <normaloff>:/img16/nuevo.png</normaloff>:/img16/nuevo.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Licencia</string>
        </attribute>
        <layout class="QVBoxLayout">
         <property name="margin">
          <number>4</number>
         </property>
         <item>
          <widget class="QTextEdit" name="txtLicense">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnAcercaQT">
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Acerca Qt</string>
       </property>
       <property name="icon">
        <iconset resource="../res/gr-lida.qrc">
         <normaloff>:/img16/qt.png</normaloff>:/img16/qt.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>362</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Aceptar</string>
       </property>
       <property name="icon">
        <iconset resource="../res/gr-lida.qrc">
         <normaloff>:/img16/aplicar.png</normaloff>:/img16/aplicar.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabAcercaD</tabstop>
  <tabstop>txtAcercaD</tabstop>
  <tabstop>twTraductores</tabstop>
  <tabstop>txtLicense</tabstop>
  <tabstop>btnClose</tabstop>
  <tabstop>btnAcercaQT</tabstop>
 </tabstops>
 <resources>
  <include location="../res/gr-lida.qrc"/>
 </resources>
 <connections/>
</ui>
